fgrpc/pingsrv.go

```diff
@@ -28,6 +28,7 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	_ "google.golang.org/grpc/encoding/gzip" // Install the gzip compressor
```

Comment:
hello @ldemailly , here is change the server side.  refer [here](https://github.com/grpc/grpc-go/blob/master/examples/features/compression/server/main.go#L30)